extras/create_send_notifications.py

```diff
@@ -73,7 +86,7 @@ def generate_notification_pdfs(db, org_ids, master_report_key):
         if was_encrypted:
             num_pdfs_created += 1
             logging.info("{} - Created encrypted notification PDF".format(org_id))
-        elif results is not None and len(results["notifications"]) == 0:
+        elif results is not None and len(results["notifications"]) == 0: 
```

Comment:
No need for a trailing space here:
```suggestion
        elif results is not None and len(results["notifications"]) == 0:
```